### PR TITLE
Change the github action instructions, so that it doesn't show a spec…

### PR DESCRIPTION
**Pull request by Lucais11**
…ific access token(which was confusing customers)

### Summary of changes
Made a specific small change in the Continuous Deployment doc that shows an access token, which made customer's think they need to use that specific token, instead of generating one themselves. Link to doc- https://fly.io/docs/app-guides/continuous-deployment-with-github-actions/ 
### Related GitHub and Fly.io community links
#1065

### Notes
n/a if none

### CODE REVIEW

File: app-guides/continuous-deployment-with-github-actions.html.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The first section is a speed-run through the steps to make the go-example app au
 1.  Fork the [go-example](https://github.com/fly-apps/go-example) repository to your GitHub account.
 2.  Clone the new repository to your local machine.
 3.  Run `fly launch` from within the project source directory to create a new app and a `fly.toml` configuration file. Type `N` when `fly launch` asks if you want to set up databases and `N` when it asks if you want to deploy.
-4.  Still in the project source directory, get a Fly API deploy token by running `fly tokens create deploy -x 999999h`. Copy the output.
+4.  Still in the project source directory, get a Fly API deploy token by running `fly tokens create deploy -x “custom access token”`. The access token can be generated at https://fly.io/user/personal_access_tokens. Copy the output.
```
Comment (catflydotio) [inline app-guides/continuous-deployment-with-github-actions.html.md:27]:
```suggestion
4.  Still in the project source directory, get a Fly API deploy token by running `fly tokens create deploy -x <your-access-token>`. The access token can be generated at https://fly.io/user/personal_access_tokens. Copy the output.
```

Our convention is to put angle brackets around "variables" in CLI examples.

This suggestion is based on the assumption that you don't actually need double quotes around the token string -- if you do, we need to add the double quotes back in.

Comment (anatoliykmetyuk) [reply app-guides/continuous-deployment-with-github-actions.html.md:27]: Oh, I thought `9999999h` was the lifetime of the token, i.e. "expire in that many hours".

But it's still not clear to me why run `fly tokens create deploy -x “custom access token”` with the custom token at all? For me, the deployment worked entirely without running that command, I just generated the token at the dashboard and inserted it directly into GH repo secrets. What is the purpose of this command?

Comment (andie787) [reply app-guides/continuous-deployment-with-github-actions.html.md:27]: hi @anatoliykmetyuk 
You're right that the value of `-x` is the token expiry!

Sorry, talked about this internally with @Lucais11 yesterday, but we didn't get back to this PR yet to fix it.

(Per the discussion over in #1065, we weren't sure if you got the error when you ran the `fly tokens create deploy` command or when you tried to use the token you got from that command. Since the error happened when you tried to use the token, then we'll have to look into what happened there.)

The short version is that you solved your issue by using a different kind of token.

Comment (andie787) [reply app-guides/continuous-deployment-with-github-actions.html.md:27]: I realized I didn't answer all your questions. Just to clarify, you would not run `fly tokens create deploy -x “custom access token”`, that was a misunderstanding. 
The command `fly tokens create deploy` creates a deploy token. A deploy token is scoped to the app you created it for, while the token you created in the UI under **Account** is a user-scoped OAuth token.

Comment (anatoliykmetyuk) [reply app-guides/continuous-deployment-with-github-actions.html.md:27]: > we weren't sure if you got the error when you ran the fly tokens create deploy command or when you tried to use the token you got from that command. Since the error happened when you tried to use the token, then we'll have to look into what happened there.) The short version is that you solved your issue by using a different kind of token

Yes, to clarify, `fly tokens create deploy` worked just fine, but the token it produced did not work when I was actually deploying from GH Actions.

**Comment by andie787** [comment]
@Lucais11 I think we can close this now?
